packages/suite-desktop-core/src/modules/bluetooth.ts

```diff
@@ -1,32 +1,76 @@
 import { ipcMain } from 'electron';
 
 import { IpcProxyHandlerOptions, createIpcProxyHandler } from '@trezor/ipc-proxy';
+import { getFreePort } from '@trezor/node-utils';
 import { BluetoothIpc, BluetoothIpcApi } from '@trezor/transport-bluetooth';
 
+import { BluetoothProcess } from '../libs/processes/BluetoothProcess';
+
 import type { ModuleInit } from './index';
 
 export const SERVICE_NAME = '@trezor/transport-bluetooth';
 
 export const init: ModuleInit = () => {
     const { logger } = global;
 
+    let bluetoothProcess: BluetoothProcess | undefined;
+
+    const getBluetoothProcess = async () => {
+        if (!bluetoothProcess) {
```

Comment:
```suggestion
        if (bluetoothProcess === undefined) {
```

:pray: To avoid implicit conversion. 

This applies to all other places where is implicit conversion to `bool` from `undefined`

Reply:
i dont think implicit conversion is a issue here. i dont like this sintax so im leaving as is